src/components/HotRepoCard.tsx

```diff
@@ -26,8 +29,19 @@ const HotRepoCard = ({ repoName }: HotRepoCardProps): JSX.Element => {
 
   if (isError) {
     return (
-      <div className="p-4 border rounded-2xl bg-white w-full space-y-1 relative">
-        {`${repoName} failed to load`}
+      <div className="p-4 border rounded-2xl bg-white w-full space-y-1 relative flex flex-col justify-between">
+        {`⚠️ ${repoName} repo could not be loaded`}
+
+        <div className="flex justify-center">
+          <a
+            className="bg-cheesyYellow text-grey rounded-xl font-bold hover:text-saucyRed transition-all duration-300 mr-3 p-2 flex w-5/6 h-fit justify-center"
+            href={`${String(bugReportLink)}`}
```

Comment:
This could be improved but the gist of it is to pre-complete the title with the repo that doesn't work! 🍕 

```
href={`${String(`${bugReportLink}%20%{encodeURI(repoName)}%20could%20not%20be%20loaded`)}`}
```

Reply:
`encodeURI` is still needed for dealing with spaces, then it's good to go

Reply:
![image](https://user-images.githubusercontent.com/37635083/197072583-50cdc5fb-04b0-44c8-8401-008420f205da.png)
@0-vortex I don't understand what you mean, I putted here an screenshot

Reply:
`encodeURI` is changing spaces into `%20` - not a hard stop but need to merge with beta to push this forward